public/doc/swagger-2.yaml

```diff
@@ -226,8 +269,14 @@ paths:
             type: array
             items:
               $ref: '#/definitions/ServicePlan'
-        '400':
-          description: bad input parameter
+        '401':
+          $ref: '#/responses/Unauthorized'
+        '403':
+          $ref: '#/responses/Forbidden'
+        '404':
+          description: Portfolio Item Not found
+        '500':
```

Comment:
Would a 503 `:service_unavailable` make more sense?

Reply:
@syncrou 500 errors are debatable if there is a stack trace in our code or we can't access the external service

Reply:
@syncrou Rails converts any unhandled exception to 500

Reply:
@mkanoor - I was debating the value of documenting a 500 since it seems to never be a desirable response.  Though, I can also see that if a customer runs into a 500 it might be nice to know they ran into an unhandled exception.

Should our Swagger Doc include a generic 503 documentation for when our service is down?